vortex-array/src/arrays/primitive/compute/between.rs

```diff
@@ -22,6 +23,16 @@ impl BetweenFn<&PrimitiveArray> for PrimitiveEncoding {
         let nullability =
             arr.dtype.nullability() | lower.dtype().nullability() | upper.dtype().nullability();
 
+        if lower.is_null() || upper.is_null() {
```

Comment:
Shouldn’t this be in entry point function, seems pretty universal?

Reply:
sounds good